common/defs/ethereum/networks.json

```diff
@@ -9,35 +9,27 @@
         "slip44": 60
     },
     {
-        "chain": "rop",
-        "chain_id": 3,
+        "chain": "hol",
+        "chain_id": 17000,
         "is_testnet": true,
-        "name": "Ropsten",
```

Comment:
Ropsten and Rinkeby are both mentioned in `sign_tx_eip155.json` file

Reply:
c2f710a82ef99da500ce33d57cd569acd288c802